<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Face_judg</class>
 <widget class="QWidget" name="Face_judg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="lcdBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{spacing: 130px;font: 18px;}</string>
   </property>
   <property name="text">
    <string notr="true">1、表的显示屏界面是否显示完整</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>100</number>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sureButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="supEdit1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>人工待确认项</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="supEdit2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="alarmBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{spacing: 125px;font: 18px;}</string>
   </property>
   <property name="text">
    <string>3、表的预警、告警状态是否无误 </string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="supBox2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>430</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>需额外补充：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择即确认无误</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="keyBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{spacing: 180px;font: 18px;}</string>
   </property>
   <property name="text">
    <string notr="true">2、表的按键是否功能无误 </string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="testBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{spacing: 220px;font: 18px;}</string>
   </property>
   <property name="text">
    <string>4、极性测试是否通过</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="supBox1">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
